src/itemset.js

```diff
@@ -115,11 +115,13 @@ define([
                     var possibleSrcs = _.map(datasources.getPossibleFixtures(), 
                                              function(val) { return val.src; }),
                         notCustom = _.contains(possibleSrcs, itemsetData.instance.src),
-                        choices = datasources.autocompleteChoices(itemsetData.instance.src);
+                        choices = datasources.autocompleteChoices(itemsetData.instance.src),
+                        strippedValue = itemsetData.valueRef.replace(/(\[.+])+/, ""),
+                        strippedLabel = itemsetData.labelRef.replace(/(\[.+])+/, "");
```

Comment:
Err crap I think I responded to @millerdev incorrectly anyways.

This is only for the value and label in itemsets, so it doesn't handle instances. https://github.com/dimagi/Vellum/blob/master/tests/static/itemset/dropdown-fixture.xml#L26-L27

So the form there should be attribute[filter][filter]. Is there another use case of value/label i should be thinking of?